### DISCUSSION TITLE
INFO: Waiting load, interval: 500 ms, percentage: 0%

### DISCUSSION THREAD

**Discussion by yonglie** [milvus-io/milvus#15830] [Q&A and General discussion]
我用java的api调用milvus2.0.1
load集合collection的时候，有的时候卡住了，有的时候加载了就显示INFO: Waiting load, interval: 500 ms, percentage: 0%

然后查询也什么都查不到，这是为什么呢？

我用那个工具attu也试了，也是一样的卡住，有时候我插入数据了，然后那个工具里面还是没有显示刚刚插入的条数。

**Comment by yhmo** [comment]
去issue里面开一个issue，把milvus容器的日志贴上去，会有人处理的。

**Comment by yonglie** [comment]
您好，找到这个的答案了，是因为释放了collection的链接，因为是单利的，所以不能释放。


我现在又有个新的问题了，我用VGG16做人脸搜索，4096维度的，但是发现搜不到正确的结果，
但是我用程序里面自带的找距离的算法，就可以找到，是不是跟维度有关呢？








------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "milvus-io/milvus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月3日(星期四) 上午9:53
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [milvus-io/milvus] INFO: Waiting load, interval: 500 ms, percentage: 0% (Discussion #15830)





 
去issue里面开一个issue，把milvus容器的日志贴上去，会有人处理的。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by yhmo** [reply]
> 您好，找到这个的答案了，是因为释放了collection的链接，因为是单利的，所以不能释放。 我现在又有个新的问题了，我用VGG16做人脸搜索，4096维度的，但是发现搜不到正确的结果， 但是我用程序里面自带的找距离的算法，就可以找到，是不是跟维度有关呢？
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "milvus-io/milvus" ***@***.***&gt;; 发送时间:&nbsp;2022年3月3日(星期四) 上午9:53 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [milvus-io/milvus] INFO: Waiting load, interval: 500 ms, percentage: 0% (Discussion #15830) 去issue里面开一个issue，把milvus容器的日志贴上去，会有人处理的。 — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

"释放了collection的链接"是指哪个操作？如果是 ReleaseCollection() 之后再LoadCollection()卡住，那应该算一个bug才对。

**Reply by yhmo** [reply]
查询结果有严格的测试用例来保证正确性，一般来说不会出问题。先做一个简单排查：如果插入10条4096维度的向量，然后用milvus搜索出来的结果是否正确？

**Comment by yonglie** [comment]
您好，找到这个的答案了，是因为释放了collection的链接，因为是单利的，所以不能释放。


我现在又有个新的问题了，我用VGG16做人脸搜索，4096维度的，但是发现搜不到正确的结果，
但是我用程序里面自带的找距离的算法，就可以找到，是不是跟维度有关呢？


但是我用ResNet50的2048维度的效果还是蛮不错的，都可以搜到类别。
到底为什么呢？






------------------ 原始邮件 ------------------
发件人:                                                                                                                        "milvus-io/milvus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月3日(星期四) 上午9:53
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [milvus-io/milvus] INFO: Waiting load, interval: 500 ms, percentage: 0% (Discussion #15830)





 
去issue里面开一个issue，把milvus容器的日志贴上去，会有人处理的。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yhmo** [comment]
另外如果英文还可以的话，下次提问题尽量用英文吧，这样碰到类似问题的英文用户也能看得懂。
